lib/spack/spack/test/concretize.py

```diff
@@ -17,15 +17,19 @@
 import spack.concretize
 import spack.config
 import spack.detection
+import spack.environment as ev
 import spack.error
 import spack.hash_types as ht
 import spack.platforms
 import spack.repo
 import spack.variant as vt
 from spack.concretize import find_spec
+from spack.main import SpackCommand
 from spack.spec import CompilerSpec, Spec
 from spack.version import ver
 
+env = SpackCommand("env")
```

Comment:
Can't we test concretization without using the full `spack environment` command?